local.cfg

```diff
@@ -0,0 +1,138 @@
+# Global configuration.
+
+# Meta-project to build
+PROJECT=deal.II-toolchain
+
+# Option {ON|OFF}: Use fresh build directory by remove existing ones?
+CLEAN_BUILD=OFF
+
+#########################################################################
+
+# Where do you want the downloaded source files to go?
+DOWNLOAD_PATH=${PREFIX_PATH}/src
+
+# Where do you want to unpack source files to go?
+#UNPACK_PATH=${PREFIX_PATH}/tmp/unpack
+
+# Where do you want the build files to go?
+#BUILD_PATH=${PREFIX_PATH}/tmp/build
+
+# Where do you want the compiled software installed?
+INSTALL_PATH=${PREFIX_PATH}/install
+
+#########################################################################
+# Set up mirror server url(s), to speed up downloads, e.g.
+#   MIRROR="${MIRROR} http://server1.org/package_mirror_dir/"
+#   MIRROR="${MIRROR} ftp://server2.org/package_mirror_dir/"
+MIRROR="https://tjhei.info/candi-mirror/ https://cecas.clemson.edu/dealii/mirror/"
+
+#########################################################################
+# Choose additional configuration and components of deal.II
+DEAL_II_CONFOPTS=""
+
+# Option {ON|OFF}: Enable machine-specific optimizations (e.g. -march=native)?
+NATIVE_OPTIMIZATIONS=OFF
+
+# Option {ON|OFF}: Enable building of dealii examples?
+BUILD_EXAMPLES=ON
+
+# Option {ON|OFF}: Unset CXX and set the compiler as MPI_CXX_COMPILER when configuring deal.II
+#
+# This is recommended for very recent CMake versions but it currently is not reliable enough
+# to enable by default.
+USE_DEAL_II_CMAKE_MPI_COMPILER=OFF
+
+# Option {ON|OFF}: Run tests after installation?
+RUN_DEAL_II_TESTS=OFF
+
+# Choose the python interpreter to use. We pick python2, python3,
+# python in that order by default. If you want to override this
+# choice, uncomment the following:
+#PYTHON_INTERPRETER=python3
+
+# Now we pick the packages to install:
+PACKAGES="load:dealii-prepare"
+
+# These are system dependencies that are often pre-installed:
+#PACKAGES="${PACKAGES} once:zlib"
+#PACKAGES="${PACKAGES} once:bzip2"
+#PACKAGES="${PACKAGES} once:git"
+#PACKAGES="${PACKAGES} once:cmake"
+#PACKAGES="${PACKAGES} once:boost"
+#PACKAGES="${PACKAGES} once:numdiff"
+#PACKAGES="${PACKAGES} once:openblas"
+
+# These packages are not dependencies but are tools used for deal.II development:
+#PACKAGES="${PACKAGES} once:astyle"
+#PACKAGES="${PACKAGES} once:numdiff"
+
+# These packages determine the active components of deal.II:
+#PACKAGES="${PACKAGES} once:adolc"
+#PACKAGES="${PACKAGES} once:arpack-ng"
+#PACKAGES="${PACKAGES} once:assimp"
+#PACKAGES="${PACKAGES} once:ginkgo"
+#PACKAGES="${PACKAGES} once:gmsh"
+#PACKAGES="${PACKAGES} once:gsl"
+#PACKAGES="${PACKAGES} once:opencascade"
+PACKAGES="${PACKAGES} once:parmetis"
+#PACKAGES="${PACKAGES} once:sundials"
+#PACKAGES="${PACKAGES} once:superlu_dist"
+PACKAGES="${PACKAGES} once:hdf5"
+#PACKAGES="${PACKAGES} once:netcdf"
+PACKAGES="${PACKAGES} once:p4est"
+PACKAGES="${PACKAGES} once:trilinos"
```

Comment:
If you use the local.cfg for testing, the shouldn't you activate parmetis, superlu_dist, trilinos and of course dealii?

Reply:
superlu didn't work until a recent patch